src/sdk/namespace_s3.js

```diff
@@ -692,7 +700,13 @@ class NamespaceS3 {
 
     /**
      * 
-     * @param {AWS.S3.HeadObjectOutput | AWS.S3.GetObjectOutput} res 
+     * @param {Omit<Partial<
```

Comment:
`Omit` is from typescript? (it is a javascript file I don't understand).
And another question - did you use an automatic tool for all the changes that are JSDoc comments?

Reply:
This is a jsdoc with typescript.
https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html#supported-jsdoc

Reply:
the tool is tsc